pkg/docker/containers-view.jsx

```diff
@@ -119,7 +121,8 @@ var ContainerDetails = React.createClass({
             <div className='listing-ct-body'>
                 <dl>
                     <dt>{_("Id")}      </dt> <dd>{ container.Id }</dd>
-                    <dt>{_("Created")} </dt> <dd>{ container.Created }</dd>
+                    <dt>{_("Created")} </dt>
+                    <dd>{ moment(container.Created).isValid() ? moment(container.Created).calendar() : container.Created }</dd>
```

Comment:
`image.Created` is an Unix time stamp. But the code here looks like `container.Created` is a string, as it previously was shown verbatim. Could the code be fixed to not convert it to a string in the first place, and only store it as a proper timestamp? Or are the time strings already coming from teh docker REST API, and images' is converted elsewhere? (e. g. with `moment.unix()`, but I don't see where this is happening)

Reply:
The time string is coming from the REST API. However 2 calls are being made, one to get a list of all containers and one to get more detailed information on a specific container.

The first call has got a unix timestamp, the second is (peculiarly) a string. We could overwrite the string from the second call with the unix timestamp we got from the first call.

Reply:
Unless it's too hacky, I think it is generally preferrable to handle unix timestamps everywhere in the data model, and just convert them to strings on presentation. The docker code already has various back and forth conversions to work around that - this now also clarifies why it's doing that.

So I think it's fine to keep this change as-is for now, as it's defensive enough. Cleaning up the code to use timestamps can be a follow-up PR.